lib/ProteinStructureUtils/Utils/PDBUtils.py

```diff
@@ -74,15 +98,103 @@ def _file_to_data(self, file_path):
             'num_chains': chain_no,
             'num_residues': res_no,
             'num_atoms': atom_no,
-            'protein': {
+            'proteins': [{
                 'id': os.path.basename(file_path),
                 'sequence': seq,
                 'md5': hashlib.md5(seq.encode()).hexdigest()
-            },
+            }],
         }
 
         return data, pp_no
 
+    def _exp_file_to_data(self, file_path):
+        """
+        _exp_file_to_data:
+            Do the PDB conversion--parse the experiment pdb file for creating a pdb data object
+        """
+        # TODO: Figure out how to parse the experimental pdb file for an experimental data structure
+        parser = PDB.MMCIFParser()
+        cif = file_path
+        structure = parser.get_structure("PHA-L", cif)
+        #structure.header = structure.header
+        #print(f'The mmcif structure header for {cif}*************************:\n\n')
+        #kwds = structure.header.get('keywords')
+        #print(f'Keywords: {kwds}')
+        #resl = structure.header.get('resolution')
+        #print(f'Resolution: {resl}')
+
+        # create a dictionary that maps all mmCIF tags in an mmCIF file to their values
+        #mmcif_dict = MMCIF2Dict(cif)  # TypeError: 'module' object is not callable
+        #print("The mmcif dictionary:*****************************************\n\n")
+        # E.g., get the solvent content from an mmCIF file:
+        #sc = mmcif_dict["_exptl_crystal.density_percent_sol"]
+        #print(f'_exptl_crystal.density_percent_sol: {sc}')
+        # get the list of the y coordinates of all atoms
+        #y_list = mmcif_dict["_atom_site.Cartn_y"]
+        # print(f'_atom_site.Cartn_y: {y_list}')
+
+        ppb = PPBuilder()
+        pp_list = []
+        pp_no = 0
+        for pp in ppb.build_peptides(structure):
+            pp_no += 1
+            my_seq = pp.get_sequence()
+            pp_list += str(my_seq)
+        seq = ''.join(pp_list)
+
+        cpd_dict = dict()
+        cpd = structure.header.get('compound', {})
+        print(f'Compound:\n {cpd}')
+        if cpd and cpd.get('1'):
+            cpd_dict = cpd.get('1')
+
+        src_dict = dict()
+        src = structure.header.get('source', {})
+        if src and src.get('1'):
+            src_dict = src.get('1')
+
+        hd = self._upload_to_shock(file_path)
+
+        mmcif_data = {
```

Comment:
make sure your `mmcif_data` matches what you defined in `ExperimentalProteinStructure`.